game/theater/start_generator.py

```diff
@@ -625,6 +628,12 @@ def generate_ground_points(self) -> None:
                     skip_sams -= 1
                 else:
                     self.generate_aa_site()
+            # 1 in 4 additional objectives are EWR.
+            if random.randint(0, 3) == 0:
```

Comment:
`elif`. Otherwise it's possible for the same iteration from L624 to generate both an EWR and a SAM, which makes the count on L623 wrong.

Could be worth rebalancing the quantities and odds here, but that's probably better suited to a follow up.

Reply:
Done.  I agree, the quantities and odds likely will need to be balanced as well.